File: PJDesignBackStageAPI/App.Model/Category/CreateCategory.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class CreateCategoryRequest
    {
        public int UnitId { get; set; }
        public string Name { get; set; } = null!;
        public bool IsEnabled { get; set; }
    }
}

File: PJDesignBackStageClient/src/app/shared/models/create-category.ts
export class CreateCategoryRequest {
  unitId: number;
  name: string;
  isEnabled: boolean;

  constructor(unitId: number, name: string, isEnabled: boolean) {
    this.unitId = unitId;
    this.name = name;
    this.isEnabled = isEnabled;
  }
}
